LeagueSandbox-Default.csproj

```diff
@@ -109,5 +103,8 @@
       <Name>GameServerLib</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <None Include="app.config" />

```

Comment:
What does this actually include?

Reply:
I don't really know, that was probably added by Visual Studio and/or Resharper automatically. That file doesn't exist either. All I did was remove unused refs.

Reply:
There's one over at GameServer (https://github.com/LeagueSandbox/GameServer/blob/a7c76889a942b89d22bb55210eb5ebfb28f15672/GameServerApp/App.config)

Maybe this addition should be removed from here.

Reply:
It seems like app.config has been .gitignored

Reply:
https://github.com/LeagueSandbox/LeagueSandbox-Default/commit/61efe42c59491348a683882e8a8c3c49cab186b1 I have ignored app.config 🤔